src/reducers/index.js

```diff
@@ -1,11 +1,63 @@
 import { combineReducers } from 'redux';
+import _ from 'lodash';
 import { reducer as formReducer } from 'redux-form';
 import { routerReducer } from 'react-router-redux';
+import { blur } from 'redux-form';
 
 import selectionReducer from 'src/reducers/selectionReducer';
+import { socialSecurity, phoneNumber } from 'src/helpers/fieldNames';
+
+const normalizePhoneNumber = input => {
```

Comment:
I do feel that this could be improved.  Switching to the `substr()` approach is a good option, but I'd like to explore ways to do it even more efficiently.  I really think that there's a way to do that.